src/libsync/clientsideencryptionjobs.cpp

```diff
@@ -35,6 +35,7 @@ void GetFolderEncryptStatusJob::start()
 	req.setPriority(QNetworkRequest::HighPriority);
 	req.setRawHeader("OCS-APIREQUEST", "true");
     req.setHeader(QNetworkRequest::ContentTypeHeader, QByteArrayLiteral("application/xml"));
+    req.setRawHeader("Depth", "infinity");
 
 	QByteArray xml = R"(<d:propfind xmlns:d="DAV:"> <d:prop xmlns:nc="http://nextcloud.org/ns"> <nc:is-encrypted/> </d:prop> </d:propfind>)";
 	auto *buf = new QBuffer(this);
```

Comment:
Yes... the E2EE code has an interesting mix of 2 spaces, 4 spaces and tabs (*gasp*) indents...

I'm going the road of fixing it one line at a time instead of running clang-format on it. That explains what you see and I hate it.